cli/serverless/golang/templates/main.tmpl

```diff
@@ -4,6 +4,12 @@ func InputSchema() any {
 }
 {{end}}
 
+{{if and .WithDescription (not .WithDataTags)}}
```

Comment:
Simply determine `{{if .WithDataTags}}`. The default behavior should be passing empty tag list to server, and this kind of logic is the responsibility of server-side.